### ISSUE TITLE
SickRage not updating automatically

### ISSUE THREAD

**Issue by rmbleeker** [SickRage/SickRage#1737]
**Branch:** master
**Commit:** d9d9ead0938a8e736a35aa45e1f37147dd0cdf49
**Version:** 2016.04.17-2
**Database Version:** 43.1 
**OS:** Linux-4.2.0-35-generic-x86_64-with-Ubuntu-14.04-trusty

**What you did:** A manual check for updates after finding out SickRage hasn't updated itself in 2 weeks, only to be told "no update needed".

**What happened:** The last time I looked at SickRage I noticed it hadn't updated in a week or so. A manual check for updates said "no update needed". I then performed a [Git reset](https://github.com/SickRage/SickRage/wiki/FAQ's-and-Fixes#i-have-problems-updating-or-my-installation-got-corrupt-what-now) and it pulled the latest changes, so I assumed everything was fine again. Now 2 weeks later SickRage is again not up-to-date, I again get the message "no update needed" when manually checking for updates. I will hold off on another Git reset until one of the devs tells me to do one.

The update interval of SickRage is set to 6 hours, so it should pull the latest version 4 times a day.
![screenshot from 2016-05-10 11 09 19](https://cloud.githubusercontent.com/assets/4312801/15141448/5321793c-16a0-11e6-9539-5ca7263b483a.png)

I'm on the master branch and have entered my Git credentials even though that's not necessary to receive updates.
![screenshot from 2016-05-10 11 09 52](https://cloud.githubusercontent.com/assets/4312801/15141479/7b3fe192-16a0-11e6-92ea-44a231c2bb08.png)

**What you expected:** SickRage to update automatically.

**Logs:** I can't find anything in the logs about automatic updates, but whenever I do a manual update I see this:
```
2016-05-10 10:44:58 INFO     Thread-16 :: No update needed
```

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [x] Post debug logs, either inline (for smaller logs) or using gist
- [x] Please wrap inline logs inside \`\`\` and \`\`\` for readability

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by miigotu** [comment]
There havent been any updates for 2 weeks, until today =P

**Comment by rmbleeker** [comment]
Had not expected that to be honest. Is it also the case that there were longer update intervals before that, like the week before? 

I'll wait and see then, I guess I should have this update automatically applied later today. It hasn't so far though, a check for updates still says "no update needed".

**Comment by miigotu** [comment]
I just started a new job about a month ago, and we have been delaying merges to master while the responsive-ui is being worked on, so yeah.

**Comment by rmbleeker** [comment]
I again noticed SickRage hadn't send me any update notification for a while, I use Telegram for SR notifications. So I've been digging a little, even though I don't know a lot about git. But I found something that looks kind of like an update log in the file ~/.git/logs/HEAD in the home directory of the `sickrage` user. Here is the relevant part, which is the last 11 lines of the file minus the commit hashes:

```
1460800028 +0200     pull -f origin master: Fast-forward
1460847816 +0200     checkout: moving from master to master
1460847848 +0200     reset: moving to upstream/master
1460847865 +0200     checkout: moving from master to master
1461253077 +0200     checkout: moving from master to master
1461253078 +0200     reset: moving to upstream/master
1461253149 +0200     checkout: moving from master to master
1461573928 +0200     checkout: moving from master to master
1467254987 +0200     checkout: moving from master to master
1467254989 +0200     reset: moving to upstream/master
1467255089 +0200     checkout: moving from master to master
```
Now if you were to convert those Unix timestamps to regular dates, you'll see that the last `pull` line was added on April 16th, which incidentally is also the last time SR send an update notification to Telegram. The timestamps of the other lines are spread out over 4 dates: April 16 again, April 21, April 25 and June 30th. On each of those dates I performed a git reset of SickRage because I suspected something was wrong, and on each of those dates I pulled in a couple of commits. I was 99 commits behind on June 30th for example, which was last night. So there were updates for SickRage available but I wasn't getting them. Something is clearly wrong with the auto update function of my SickRage installation, I just can't figure out what. The update settings are still as shown in the screenshot in my original post. Any help or clarification would be appreciated.
